include/range/v3/view/zip_with.hpp

```diff
@@ -321,8 +321,8 @@ namespace ranges
             {
                 template<typename Fun, typename ...Rngs>
                 using Concept = meta::and_<
-                    meta::and_<InputRange<Rngs>...>,
-                    CopyConstructible<Fun>(),
+                    InputRange<Rngs>...,
```

Comment:
Funny, I'm certain there was a reason I needed the inner `meta::and_`, but from the passing tests it seems that it's no longer needed.